.github/workflows/release.yaml

```diff
@@ -1,27 +1,98 @@
-name: Release Charts
+name: Release Docker
 
 on:
   push:
+    tags: 
+      - 'v*.*.*'
     branches:
-      - master
+      - 'master'
 
+env:
+  REGISTRY: quay.io
+  IMAGE_NAME: prometheusmsteams/prometheus-msteams
+  
 jobs:
-  release:
+  build:
+
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Docker buildx
+        uses: docker/setup-buildx-action@79abd3f86f79a9d68a23c75a09a9a85889262adf
```

Comment:
should we use `v2` instead of the commit hash?

Reply:
Good point. I wanted to address this with automatic updates. Let's move discussion to #258